integration_tests/test_systems_real_api.py

```diff
@@ -0,0 +1,199 @@
+import json
+
+import docker
+import jcapiv1
+import os
+
+from click.testing import CliRunner
+from jcapiv1 import SystemsApi
+from integration_tests import API_KEY
+from jccli import cli
+
+
+JC_CONNECT_KEY = os.getenv('JC_CONNECT_KEY')
+DOCKER_IMAGE = 'sagejcdevs/jumpcloud-agent:v0.0.2'  # Name of image to spin up
+SYSTEM_COUNT = 5  # The number of docker containers to spin up and connect to the cloud
+
+
+class TestSystemsRealApi:
+    @classmethod
+    def setup_class(cls):
+        '''
+        Spin up docker images and check that they have connected
+        '''
+
+        apiv1_configuration = jcapiv1.Configuration()
+        apiv1_configuration.api_key['x-api-key'] = API_KEY
+        cls.systems_api = SystemsApi(jcapiv1.ApiClient(configuration=apiv1_configuration))
+
+        current_systems = cls.systems_api.systems_list(
+            content_type='application_json',
+            accept='application/json'
+        )
+        assert current_systems.total_count == 0
+
+        docker_client = docker.from_env()
+        cls.containers = []
+        for i in range(SYSTEM_COUNT):
+            container = docker_client.containers.run(
+                DOCKER_IMAGE,
+                environment={'JC_CONNECT_KEY': JC_CONNECT_KEY},
+                detach=True,
+                stdin_open=True,
+                tty=True
+            )
+            cls.containers.append(container)
+
+
+        # Wait for containers to boot up and connect to JumpCloud
+        while True:
+            current_systems = cls.systems_api.systems_list(
+                content_type='application/json',
+                accept='application/json'
+            )
+            print("number of reporting systems on JumpCloud:")
+            print(current_systems.total_count)
+            if current_systems.total_count == SYSTEM_COUNT:
```

Comment:
I don't think a client can choose a particular group to be added to when they activate the JC agent (I could be wrong about that though).

I wrote all these integration tests for running on a clean JC instance; I think if we want to do anything else I will need to talk to you about how to do it.

Reply:
here's the API on adding/removing systems to/from a `systems group`:
https://docs.jumpcloud.com/2.0/system-group-members-and-membership/list-the-members-of-a-system-group

Take a look at the `op`, it has two operations `add` and `remove`.  It would be better to put all these systems in a special testing group.  Can do in a follow up PR though.